pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -262,6 +263,61 @@ func (b *OFBridge) IsConnected() bool {
 	return b.ofSwitch.IsReady()
 }
 
+func (b *OFBridge) AddFlowsInBundle(flows []Flow, modify bool) error {
```

Comment:
As it is a flag to indicate it is a new flow or not, but not to indicate the operation is "modify", I feel better to rename it like: newFlows, isModification, etc..